### PR TITLE
MIAJS-1306 - Print map doc

### PR DESCRIPTION
**Pull request by simonlaustsen**
I've added a few lines about this feature and attached some screenshots.
You are welcome to add more if you like :)

### CODE REVIEW

File: collections/_various/map_print.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+title: Web App - Print map
+nav_weight: 1300
+published: true
+date: 2020-01-31
+permalink: /print-map/
+---
+
+## Overview
+
+The print map tool can be used for different purposes fx. to print evacuation routes for each floor at your venue.
```
Comment (marksdk) [inline collections/_various/map_print.md:11]:
```suggestion
The print map tool is found in the Standard Web App and can be used for different purposes, e.g. to print evacuation routes for each Floor at your Venue.
```

File: collections/_various/map_print.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+The print preview can be triggered by the print button at top right or by one of the following shortcuts:
```
Comment (marksdk) [inline collections/_various/map_print.md:13]:
```suggestion
The print preview can be triggered by the print button in the top right corner of the map view in the Standard Web App, or by one of the following shortcuts:
```

File: collections/_various/map_print.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+* CMD + P (macOS)
+* CTRL + P (Windows)
+
+![Map Print Feature]({{ site.url }}/assets/various/map-print_button.png)
+
+which triggers a full screen map print preview as seen below
```
Comment (marksdk) [inline collections/_various/map_print.md:20]:
```suggestion
Clicking the button, or hitting the keyboard combination, triggers a full screen map print preview as seen here:
```

File: collections/_various/map_print.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+title: Web App - Print map
+nav_weight: 1300
+published: true
+date: 2020-01-31
```
Comment (marksdk) [inline collections/_various/map_print.md:5]:
```suggestion
date: 2020-11-06
```

File: collections/_various/map_print.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+title: Web App - Print map
```
Comment (marksdk) [inline collections/_various/map_print.md:2]:
```suggestion
title: Print map
```

**Review by marksdk — CHANGES_REQUESTED** [review]

**Comment by simonlaustsen** [comment]
All comments resolved :)

**Review by marksdk — APPROVED** [review]

**Pull request merged**